.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: ci
```

Comment:
Very nice @Tpt. How difficult would it be to add code-coverage?

Reply:
Thank you! GitHub does not provide a built-in interface for code coverage so we would need to choose where to upload the code coverage data (a github-hosted website, an external system, a simple %age uploaded to the MR as a comment...). I would prefer to do that in an other PR.

Reply:
Thanks! No pressure whatsoever.